cmd/cluster/kubevirt/create.go

```diff
@@ -104,5 +99,15 @@ func applyPlatformSpecificsValues(ctx context.Context, exampleOptions *apifixtur
 		RootVolumeStorageClass:    opts.KubevirtPlatform.RootVolumeStorageClass,
 		RootVolumeAccessModes:     opts.KubevirtPlatform.RootVolumeAccessModes,
 	}
+
+	if opts.BaseDomain != "" {
+		exampleOptions.BaseDomain = opts.BaseDomain
+	} else {
+		if exampleOptions.Annotations == nil {
+			exampleOptions.Annotations = map[string]string{}
+		}
+		exampleOptions.Annotations[hyperv1.KubeVirtBaseDomainPassthroughAnnotation] = "true"
```

Comment:
We're using an annotation here to signal to the controllers that automatic basedomain passthrough is in use. I picked an annotation here instead of using a new API field for this signal because I'm not yet ready to officially say "how" we'll enable this feature.

Reply:
I'm not sure that annotation effectively give us anything else than opaqueness https://github.com/openshift/hypershift/pull/1787/files#r1067315404. I'm +1 to field.